interfaces/seccomp/backend_test.go

```diff
@@ -645,16 +645,26 @@ fi`)
 	err = s.Backend.Initialize()
 	c.Assert(err, IsNil)
 
+	c.Check(s.snapSeccomp.Calls(), HasLen, 2)
+	c.Check(s.snapSeccomp.Calls(), DeepEquals, [][]string{
+		// previous compilation
+		{"snap-seccomp", "compile", profile + ".src", profile + ".bin"},
+		// initialization with new version
+		{"snap-seccomp", "version-info"},
+	})
+
 	// the profile should be rebuilt now
 	err = s.Backend.Setup(snapInfo, interfaces.ConfinementOptions{}, s.Repo)
 	c.Assert(err, IsNil)
 	c.Check(profile+".src", testutil.FileEquals, updatedProfileHeader+"\ndefault\n")
 
 	c.Check(s.snapSeccomp.Calls(), HasLen, 3)
 	c.Check(s.snapSeccomp.Calls(), DeepEquals, [][]string{
+		// previous compilation
 		{"snap-seccomp", "compile", profile + ".src", profile + ".bin"},
 		// initialization with new version
 		{"snap-seccomp", "version-info"},
+		// compilation of profiles with new compiler version
```

Comment:
Thank you for the added comments; sorry for being nitpicky, but can you be a little more descriptive with something like:

```
    // compilation from this first Setup()
633 c.Check(s.snapSeccomp.Calls(), HasLen, 1)
...
648 c.Check(s.snapSeccomp.Calls(), HasLen, 2)
649 c.Check(s.snapSeccomp.Calls(), DeepEquals, [][]string{
650	// compilation from first Setup()
651	{"snap-seccomp", "compile", profile + ".src", profile + ".bin"},
652	// initialization with new version
653	{"snap-seccomp", "version-info"},
...
662 c.Check(s.snapSeccomp.Calls(), DeepEquals, [][]string{
663	// compilation from first Setup()
664	{"snap-seccomp", "compile", profile + ".src", profile + ".bin"},
665	// initialization with new version
666	{"snap-seccomp", "version-info"},
667	// compilation of profiles with new compiler version
668	{"snap-seccomp", "compile", profile + ".src", profile + ".bin"},
```